### PR TITLE
Upgrade Go version to v1.19

### PR DESCRIPTION
**Pull request by tenzen-y**
Signed-off-by: Yuki Iwai <yuki.iwai.tz@gmail.com>

I upgraded the Go version to v1.19.

### CODE REVIEW

File: pkg/client/clientset/versioned/fake/register.go [hunk 1]
```diff
@@ -35,14 +35,14 @@ var localSchemeBuilder = runtime.SchemeBuilder{
...
+//	  "k8s.io/client-go/kubernetes"
+//	  clientsetscheme "k8s.io/client-go/kubernetes/scheme"
+//	  aggregatorclientsetscheme "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/scheme"
+//	)
 //
-//   kclientset, _ := kubernetes.NewForConfig(c)
-//   _ = aggregatorclientsetscheme.AddToScheme(clientsetscheme.Scheme)
+//	kclientset, _ := kubernetes.NewForConfig(c)
+//	_ = aggregatorclientsetscheme.AddToScheme(clientsetscheme.Scheme)
 //
```
Comment (tenzen-y) [inline pkg/client/clientset/versioned/fake/register.go:46]: I ran `make fmt`.

File: .github/workflows/golangci-lint.yml [hunk 2]
```diff
@@ -5,17 +5,19 @@ on:
       - v*
     branches:
       - master
-      - main
```
Comment (tenzen-y) [inline .github/workflows/golangci-lint.yml:8]: We don't have `main` branch.

**Review by terrytangyuan — COMMENTED** [review]
Thanks!

/lgtm

**Comment by tenzen-y** [comment]
@terrytangyuan Can you give this PR an approve label?

**Review by terrytangyuan — COMMENTED** [review]
/approve

**Pull request merged**